acceptance_tests/features/steps/survey_launched.py

```diff
@@ -31,3 +32,8 @@ def send_survey_launched_msg(context):
             message=message,
             content_type='application/json',
             routing_key=Config.RABBITMQ_SURVEY_LAUNCHED_ROUTING_KEY)
+
+
+@step("the events logged for the survey launched case are {expected_event_list}")
+def check_survey_launch_event_logging(context, expected_event_list):
```

Comment:
This step also seems to be duplicated, I would suggest writing it once in an events step file